src/client/components/Editor.tsx

```diff
@@ -32,6 +34,37 @@ class Editor extends React.PureComponent<EditorProps> {
         }
     }
 
+  componentDidMount(): void {
+      // TODO: why does using gutterclick yield weird highlighting?
+      this.aceComponent.current.editor.on("gutterclick", function(e) {
+        var target = e.domEvent.target;
+
+        if (target.className.indexOf("ace_gutter-cell") == -1){
+            return;
+        }
+
+        if (!e.editor.isFocused()){
+            return;
+        }
+
+        if (e.clientX > 25 + target.getBoundingClientRect().left){
+            return;
+        }
+
+        var breakpoints = e.editor.session.getBreakpoints(row, 0);
```

Comment:
I don't think we need this

Reply:
Do you mean you don't think we need `getBreakpoints` at all? Or specifically `getBreakpoints` with the arguments `row, 0`?